src/reducers/scrollStateHelper.js

```diff
@@ -279,94 +279,14 @@ export function updateViewHeight(state, {
 
 /**
  * @param {!Object} state
- * @param {number} deltaY
- */
-export function scrollBy(state, deltaY) {
```

Comment:
Apparently it was just "Biggest change is me axe-ing this method and using scrollTo instead"
I had more detail somewhere, but don't know where...

Gist is that this method updates row heights before choosing the row we scroll to (taking into account real height of rows we scroll over), while scrollTo doesn't update heights until after choosing the row we scroll to.  I'm not sure there's enough difference there to justify keeping this extra logic around since I don't expect these delta's to be large enough to go out of the buffer unless a user is scrolling really fast, and then I'm not sure they care...